### PR TITLE
Make compatible with current alembic version

### PR DESCRIPTION
**Pull request by Bladieblah**
Hey there, I updated the module to be compatible with the current alembic version. Also switched from setup.py to poetry and fixed the tests since they broke too. I think we can replace the whole appveyor setup with a simple github action, given how old it is I'm not sure if it works anymore. Let me know what you think!

### CODE REVIEW

**Comment by jpassaro** [comment]
thank you for the PR. I'm flattered to see this behavior is desired by somebody in the wild! 

i have some feedback on the change (well-organized commits are a boss move!) but first the headline: I'm not going to merge the change on this repo. 

i haven't touched any sql database since the pandemic started, and perhaps more importantly i never actually got to using this (or really Alembic at all) in production. i have lost quite a bit of context here and i don't expect to get it back any time soon - in fact i don't expect to become an Alembic user at all any time soon. So this should really be considered a thought exercise or a POC. Not to mention, for me to merge this I'd have to get real comfortable with your rewrite of CI/CD (i.e. test it and run it a few times) and that's simply not something I can accomplish in any satisfying amount of time.

In short, i can't retain responsibility for this project at all.

If the library as revised here is useful to you, please consider maintaining it in a repo of your own! (I'd suggest not a formal GitHub fork but just pushing the branch to a new repo of your own. would also suggest following your own instincts regarding GHA.) If you do, I'll happily link to it in the README.

Pypi part is perhaps a bit trickier. I'd suggest testing CICD under an ad-hoc name; then when ready you can contact me and we can orchestrate a transfer of the pypi project (I'll read up on how to do this). Or of course you can push your fork under a new name.

**Comment by jpassaro** [comment]
thank you for the PR. I'm flattered to see this behavior is desired by somebody in the wild! 

i have some feedback on the change (well-organized) but first the headline: I'm not going to merge the change on this repo. 

i haven't touched any sql database since the pandemic started, and perhaps more importantly i never actually got to using this (or really Alembic at all) in production. i have lost quite a bit of context here and i don't expect to get it back any time soon - in fact i don't expect to become an Alembic user at all any time soon. So this should really be considered a thought exercise or a POC. Not to mention, for me to merge this I'd have to get real comfortable with your rewrite of CI/CD (i.e. test it and run it a few times) and that's simply not something I can accomplish in any satisfying amount of time.

In short, i can't retain responsibility for this project at all.

If the library as revised here is useful to you, please consider maintaining it in a repo of your own! (I'd suggest not a formal GitHub fork but just pushing the branch to a new repo of your own. would also suggest following your own instincts regarding GHA.) If you do, I'll happily link to it in the README.

if you reach a beta/MVP or otherwise want to start publishing to pypi, you can use a name of your own of course (probably the smart move when testing your CICD); butplease get in touch and we can figure out how to transfer ownership of the

**Comment by jpassaro** [comment]
sorry my previous comment was posted before finished due to fat fingers. if you're following primarily over email, make sure to click through for the full edited response. 

also i do have some feedback. given that it's not gonna be merged I'd understand if it doesn't interest you, but let me know if it does and I'll do a review pass on your change. in case your take me up on that I'll leave this PR open for the time being.

again i really appreciate your interest. i am pleasantly surprised that there is any value here!

**Comment by Bladieblah** [comment]
That all sounds good! Reviewing won't be necessary I think, I didn't make any functional changes, 90% of it is just me running the linter. The tests are working but could still use some cleanup, I'll do that when I move to the new repo then. I have a few projects on pypi already so hopefully transferring will be straightforward, I'll also check with the sqlalchemy people if they're interested in hosting the repo and pypi.
